loader/loader.go

```diff
@@ -145,6 +145,11 @@ func (w *Worker) run(ctx context.Context, fileJobQueue chan *fileJob, workerWg *
 					}
 				})
 
+				failpoint.Inject("LoadDataSlowDown", func(val failpoint.Value) {
```

Comment:
It's better to change this to `failpoint.Inject("LoadDataSlowDown", nil)`, and modify the failpoint expression to `sleep(1000)`, e.g

`GO_FAILPOINTS=github.com/pingcap/dm/LoadDataSlowDown=sleep(1000)`

Reply:
thanks, addressed